### PR TITLE
Add merge ID sets command to sdk nightly build

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ChanochShayner** [comment]
@guykeller @roysagi 
You can see that I trigger SDK nightly build -
It has passed, and you can see in the artifacts that it has the unified id set file.

**Review by roysagi — APPROVED** [review]

File: Packs/Workday/Classifiers/classifier-IAM_Sync_User_-_Workday.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
      "complex": null,
      "simple": "City"
     },
-    "Cost Center": {
+    "Cost Center Bla": {
```
Comment (guykeller) [inline Packs/Workday/Classifiers/classifier-IAM_Sync_User_-_Workday.json:13]: let's revert this change

**Review by guykeller — APPROVED** [review]

File: Packs/Workday/Classifiers/classifier-IAM_Sync_User_-_Workday.json (hunk 1 above)
Comment (ChanochShayner) [reply Packs/Workday/Classifiers/classifier-IAM_Sync_User_-_Workday.json]: done
